itchiodl/utils.py

```diff
@@ -41,6 +41,8 @@ def clean_path(path):
     """Cleans a path on windows"""
     if sys.platform in ["win32", "cygwin", "msys"]:
         path_clean = re.sub(r"[<>:|?*\"\/\\]", "-", path)
+        # This checks for strings that end in ... or similar, weird corner case that affects fewer than 0.1% of titles
```

Comment:
is this an issue? I don't want the renaming function to be overzealous, which it seems to be heading towards